<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompositionWidget</class>
 <widget class="QWidget" name="CompositionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxComp">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayouListTool">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButtonAdd">
          <property name="text">
           <string>Добавить</string>
          </property>
          <property name="icon">
           <iconset resource="modeleride.qrc">
            <normaloff>:/add</normaloff>:/add</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonEdit">
          <property name="text">
           <string>Изменить</string>
          </property>
          <property name="icon">
           <iconset resource="modeleride.qrc">
            <normaloff>:/edit</normaloff>:/edit</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonDelete">
          <property name="text">
           <string>Удалить</string>
          </property>
          <property name="icon">
           <iconset resource="modeleride.qrc">
            <normaloff>:/remove</normaloff>:/remove</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonDown">
          <property name="text">
           <string>Ниже</string>
          </property>
          <property name="icon">
           <iconset resource="modeleride.qrc">
            <normaloff>:/down</normaloff>:/down</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonUp">
          <property name="text">
           <string>Выше</string>
          </property>
          <property name="icon">
           <iconset resource="modeleride.qrc">
            <normaloff>:/up</normaloff>:/up</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacerListTool">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxInInherited">
          <property name="text">
           <string>Унаследованные</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableViewComp">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>24</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxProp">
     <property name="title">
      <string>Свойства состава</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEditInverseDesc">
        <property name="maxLength">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDestClass">
        <property name="text">
         <string>Входящий класс:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelLinkClass">
        <property name="text">
         <string>Состав:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelDirectDesc">
        <property name="text">
         <string>Описание включения:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="RTPTechGroup::TreeComboBox::TreeComboBox" name="comboBoxDestClass"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labeAlias">
        <property name="text">
         <string>Псевдоним:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEditDirectDesc">
        <property name="maxLength">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2" colspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="RTPTechGroup::TreeComboBox::TreeComboBox" name="comboBoxLinkClass"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelIsComp">
          <property name="text">
           <string>Композиция:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEditAlias">
          <property name="maxLength">
           <number>256</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="checkBoxDestIsView">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="checkBoxIsComp">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="labelDestIsView">
          <property name="text">
           <string>Представление:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelSpace">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="labelInverseDesc">
        <property name="text">
         <string>Описание вхождения:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditSrcClass"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSrcClass">
        <property name="text">
         <string>Исходный класс:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutSave">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RTPTechGroup::TreeComboBox::TreeComboBox</class>
   <extends>QComboBox</extends>
   <header>treecombobox/treecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="modeleride.qrc"/>
 </resources>
 <connections/>
</ui>
